src/components/catalog-filter/catalog-filter.html

```diff
@@ -0,0 +1,22 @@
+<pf-filter-panel config="$ctrl.config">
+  <div class="filter-panel-container">
+    <input type="text" ng-model="$ctrl.keywordFilter.value"
+           class="keyword-filter"
+           placeholder="{{$ctrl.keywordFilter.placeholder}}"
+           ng-keypress="$ctrl.onKeywordKeyPress($event)"
+           ng-disabled="$ctrl.config.totalCount <= 1"/>
```

Comment:
```html
autocorrect="off"
autocapitalize="none"
spellcheck="false"
```

I'd remove the `/>` self-closing tag since it's not needed/recommended in HTML 5. Just `<input ...>`